MBBSEmu/Util/EnumMessageEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace MBBSEmu.Util
+{
+    public enum EnumMessageEvent
+    {
+        EnableModule,
```

Comment:
Let's add some comments here about what each of these are used for: `Enables the specified module within MbbsHost`, etc.